src/test/java/lambda/part1/exercise/Lambdas01Exercise.java

```diff
@@ -27,7 +27,7 @@ public void sortPersonsByAge() {
         Arrays.sort(persons, new Comparator<Person>() {
                     @Override
                     public int compare(Person o1, Person o2) {
-                        return o2.getFirstName().compareTo(o1.getFirstName());
+                        return Comparator.<Person>comparingInt(Person::getAge).compare(o1, o2);

```

Comment:
You are creating a new `Comparator` on each `compare` here.
Use `Integer.compare`.